plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/labeling/helper/LabelCalculationHelper.xtend

```diff
@@ -116,30 +119,37 @@ class LabelCalculationHelper {
 		return dispatchDoGetText(n4SetterDeclaration.definedSetter)
 	}
 
-	// comma separated list of all contained variable names
+	/** 
+	 * comma separated list of all contained variable names
+	 */
 	def dispatch String dispatchDoGetText(ExportedVariableStatement vs) {
-		 return vs.varDecl.map[name].join(", ")
+		return vs.varDecl.map[name].join(", ")
 	}
 
 	def dispatch String dispatchDoGetText(NamedElement namedElement) {
-		 namedElement.name
+		namedElement.name
 	}
 
-	/* dispatchDoGetText types model */
-
-	// the fully qualified module specifier, e.g. mypack/MyFile
+	/** 
+	 * dispatchDoGetText types model
+	 * the fully qualified module specifier, e.g. mypack/MyFile
+	 */
 	def dispatch String dispatchDoGetText(TModule tModule) {
 		return getModuleSpecifier(tModule)
 	}
 
-	// name + optional type variables, e.g. A<T, U>
+	/** 
+	 * name + optional type variables, e.g. A<T, U>
+	 */
 	def dispatch String dispatchDoGetText(TClassifier tClassifier) {
 		return tClassifier.name + handleTypeVars(tClassifier)
 	}
 
 	def private handleTypeVars(TClassifier tClassifier) {
```

Comment:
`handleTypeVars`sounds like an action with side effect. May be rename to `getTextOfTypeVars` or something like that?